### ISSUE TITLE
Requesting support for Ruby 2.0.0

### ISSUE THREAD

**Issue by filbranden** [ruby/setup-ruby#181]
I have a gem that I maintain that still works on Ruby 2.0 and I'd like to keep it working on that version, so I'd like to request to have it available for testing in GitHub Actions.

Ruby 2.0 is the version of Ruby shipped in the CentOS 7 / RHEL 7 Linux distribution, so it's still somewhat relevant in the real world for another short while...

```
# ruby --version
ruby 2.0.0p648 (2015-12-16) [x86_64-linux]
# gem --version
2.0.14.1
```

For now, I've been using a Travis-CI workflow and Travis offers pre-built binaries for Ruby 2.0.0-p648 on Ubuntu 14.04 (https://rubies.travis-ci.org/), but my preference would be to fully migrate all of the CI to GitHub Actions completely, and having Ruby 2.0 available would definitely help with that... It's totally OK if it's only available in a limited number of platforms or versions (e.g. ubuntu-18.04 only is totally fine.)

Let me know if you need any further information from me, or if I can help in any way to get this version up in `ruby/setup-ruby`. Thanks!

**Comment by eregon** [comment]
See https://github.com/ruby/setup-ruby/issues/133#issuecomment-751346398, I think it's simply not possible to be build such old Rubies with recent compilers (and eve if they build there would likely be many issues), so Docker images seems the only way.

**Comment by eregon** [comment]
To be fair, 2.0 seems to be more reasonable than 1.8/1.9, and there is already one build of 2.1.

So if someone wants to try building them with https://github.com/ruby/ruby-builder as suggested https://github.com/ruby/setup-ruby/issues/133#issuecomment-751346398, then that could be integrated if it works well enough.

**Comment by filbranden** [comment]
@eregon Awesome! I see you merged that commit, so it's already published in ruby-builder.

So I wrote PR #182 to add it to the list here. Tested it on my gem that I wanted to test on Ruby 2.0 and confirmed the tests are running as expected. Thanks!

BTW, in https://github.com/ruby/ruby-builder/commit/9fa277367b8cd338cfc5aca4b91d854e8e0a58b2 I noticed `ruby-2.0.0-p648` is replacing `truffleruby-21.1.0`... Not sure if that was intended or not... (But maybe this is a "build once" so you don't keep older versions?) In any case, pointing it out here, in case this wasn't intended and you might want to fix it. Cheers.
